api/gradlew

```diff
@@ -148,16 +148,17 @@ if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
     # Now convert the arguments - kludge to limit ourselves to /bin/sh
     i=0
     for arg in "$@" ; do
-        CHECK=`echo "$arg"|egrep -c "$OURCYGPATTERN" -`
-        CHECK2=`echo "$arg"|egrep -c "^-"`                                 ### Determine if an option
+        CHECK=$(echo "$arg"|egrep -c "$OURCYGPATTERN" -)
+        CHECK2=$(echo "$arg"|egrep -c "^-")                                 ### Determine if an option
 
         if [ $CHECK -ne 0 ] && [ $CHECK2 -eq 0 ] ; then                    ### Added a condition
-            eval `echo args$i`=`cygpath --path --ignore --mixed "$arg"`
+            eval args$i="$(cygpath --path --ignore --mixed "$arg")"
         else
-            eval `echo args$i`="\"$arg\""
+            eval args$i="\"$arg\""
```

Comment:
Why removing the `echo`?

Reply:
I removed the `echo` because I originally changed the line to be:
`eval "$(echo args$i)"="$(cygpath --path --ignore --mixed "$arg")"`, but then got this  shellcheck warning: https://github.com/koalaman/shellcheck/wiki/SC2116, so I removed the `echo`.